WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,16 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read(Rails.root.to_s + "/app/views/search_results/wca-regulations.json")).freeze
```

Comment:
Indeed!

Regarding the exception: I don't know what exactly is called when starting a travis environment.
From the travis file I cannot see the chef scripts being used, which means `deploy.sh rebuild_regs` would not be called, and that the json file would not be built.